pylds/base.py

```diff
@@ -406,6 +407,62 @@ def fspline(positions):
 
     return fspline
 
+def fit_vector_field(filename):
+    """
+    Returns a 2-dimensional function (vector field) fitted to 
+    data located in pylds/vector_field_files/filename.hdf5.
+
+    Parameters
+    ----------
+    filename : string
+        Name of file containing data:
+        sample_time_points: 1d array,
+            time-points in a sample time-interval.
+
+        sample_coords : list of ndarrays,
+            [x,y] contain coordinates.
+            
+        vector_field_data : ndarray, len(t),
+            Array of function/vector field values.
+    
+    Returns
+    -------
+    vector_field_interpolated : function
+        returns a vector field function to be evaluated at (t, u).
+        with t a float and u (n,2)-array.
+    """
+    hf = h5py.File('pylds/vector_field_files/'+filename+'.hdf5','r')
+    #extract data
+    time = np.array(hf.get('sample_time'))
+    coords = np.array(hf.get('sample_coords'))
+    data = np.array(hf.get('vector_field_data'))
+    
+    hf.close()
+    
+    def vector_field_wrap(v, u):
+        return v(u[0],u[1])
+    
+    def vector_field_interpolated(t, u):
+        #interpolate data in time 
+        v_interp_t = lambda t: interp1d(time, data.T, kind='cubic')(t).T
```

Comment:
t in `lambda` function is not the same as `vector_field_interpolated` argument t. Could `v_interp_t` be moved outside `vector_field_interpolated` to speed things up?

Reply:
Thanks for noticing it. Will do.